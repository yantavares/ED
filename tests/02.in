10
A .
B -.
C .-
D ...
E --
L .--
R ..
S -.-
T -
! ---
0
. - . .- . .. /.. -- -. -- .-- ... -- -.- ---