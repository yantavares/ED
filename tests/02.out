ATACAR REBELDES!
* A T R C B E D L S !